executor.go

```diff
@@ -402,12 +402,7 @@ func (e *Executor) createOptionsForLogstashServiceLogScrapping(taskInfo mesos.Ta
 	for _, ignoredKey := range e.config.ServicelogIgnoreKeys {
 		values = append(values, []byte(ignoredKey))
 	}
-	filter := scraper.ValueFilter{
-		Values: values,
-	}
-	scr := &scraper.LogFmt{
-		KeyFilter: filter,
-	}
+	scr := &scraper.JSON{}
```

Comment:
Why change the default format?

Reply:
JSON is supported by default in many tools, logfmt in none.